### DISCUSSION TITLE
Is WinUI supported for Appium Inspector?

### DISCUSSION THREAD

**Discussion by Abe-Froman** [appium/appium#19507] [Q&A] [answered]
The Inspector readme says "A GUI inspector for mobile apps and more" but what is more? I cannot get it to see my running Appium server on the same machine and a quick search through the Inspector Issue board has zero results for UWP, WPF, or WinUI so I cannot tell if this is even a supported scenario.

Thanks in advance.

**Comment by KazuCocoa** [comment]
The inspector depends on `source` and `screenshot` endpoints to show the main page source screen, and some are for element attributes. So, if the driver returns this information, the inspector can show elements.

For appium-windows-driver usage, the latest appium-inspector expects to speak W3C WebDriver protocol but the WinAppDriver might not support them, so appium-windows-driver tries to convert some of, but probably not all. Thus, some behaviors might not work properly.

Attaching the appium server log would help to see possible reasons, but currently, no one has much experience and dev env to check appium-windows-driver and relevant env in our core maintainer team I think.

**Comment by jlipps** [comment, marked as answer]
The only truly supported platforms are mobile platforms. I guess the way the "and more" is intended to be parsed is like this: `A GUI inspector (for mobile apps) and more`, not `A GUI inspector (for mobile apps and more)`

**Reply by Abe-Froman** [reply]
Thanks, I suspected as much but did want to make sure so I wouldn't spend more time trying to get something working that just isn't expected to.
